src/impls.rs

```diff
@@ -467,6 +469,35 @@ pub fn deserial_map_no_length_no_order_check<R: Read, K: Deserial + Ord, V: Dese
     Ok(out)
 }
 
+/// Write a HashMap as a list of key-value pairs in to particular order, without
+/// the length information.
+pub fn serial_hashmap_no_length<'a, W: Write, K: Serial + 'a, V: Serial + 'a>(
```

Comment:
What is this lifetime here for? I can't see it used for any of the arguments.

Reply:
I copied the signature from serial_map_no_length which also have this, but I'll make sure to remove these from both

Reply:
It might have made sense at some point.